visualization/app/codeCharta/ui/fileChooser/fileChooser.e2e.ts

```diff
@@ -23,6 +24,14 @@ describe("FileChooser", () => {
 		await goto(page)
 	})
 
+	async function showErrorDialog() {
+		const dialogErrorPageObject = new DialogErrorPageObject(page)
+		const msg = await dialogErrorPageObject.getMessage()
+		expect(msg).toEqual(" file is empty or invalid")
+		await page.waitFor(2000)
+		return dialogErrorPageObject.clickOk()
+	}
```

Comment:
- Initialize page-objects in the beforeEach-block

Write a test that opens an invalid file, checks that the dialog opened, closes the dialog and then loads a file successfully.

```
public async selectFile(path: string) {
	const fileChooser = await page.waitForFileChooser()

	await fileChooser.accept([path])

	await page.waitForSelector("#loading-gif-file")
	await page.waitForSelector("#loading-gif-file", { visible: false })
}
```
it("should open an invalid file, close the dialog and open a valid file", async () => {
	await fileChooser.openFile("./app/codeCharta/assets/logo.png")
	expect(await dialogError.getMessage()).toEqual(" file is empty or invalid")
	await dialogError.clickOk()
	await fileChooser.selectFile("./app/codeCharta/assets/sample3.cc.json")
	expect(await filePanel.getSelectedName()).toEqual("sample3.cc.json")
})
```

I don't know if this will work though.

Reply:
Could you explain why you use 
`fileChooser.selectFile("./app/codeCharta/assets/sample3.cc.json")` instead of 'open()'?

Reply:
Because openFile will click on the fileChooser and open it. However, clicking on the ok-button of the error-dialog will open the filechooser already.

That's why we just have to select files now and that's why I'm not sure if we can actually test it like this.

Reply:
The suggestion worked and it tests the behaviour perfectly !